mne_bids/mne_bids.py

```diff
@@ -39,8 +39,11 @@
                  '.fif': 'Elekta', '.gz': 'Elekta', '.pdf': '4D Magnes',
                  '.ds': 'CTF'}
 
+IGNORED_CHANNELS = defaultdict(lambda: [])
+IGNORED_CHANNELS['KIT/Yokogawa'] = ['STI 014']
 
-def _channels_tsv(raw, fname, verbose):
+
+def _channels_tsv(raw, fname, manufacturer, verbose):
```

Comment:
you could simplify this by using `_parse_ext` and have the keys of `IGNORED_CHANNELS` be the file extensions. that way you don't have to introduce a new argument

Reply:
ah yeah. I wanted a way to do it without having to add an extra argument... I didn't think about doing it that way thanks!

Reply:
+1 !